test/run-spec-wasm2c.py

```diff
@@ -465,6 +508,11 @@ def main(args):
             if options.compile:
                 o_filenames.append(Compile(cc, c_filename, out_dir, includes))
 
+        cwriter.Write()
+        main_filename = utils.ChangeExt(json_file_path, '-main.c')
+        with open(main_filename, 'w') as out_main_file:
+            out_main_file.write(output.getvalue())
```

Comment:
These lines look like the just moved down?  Is that change needed?

Reply:
Yes (copying from above), we need wasm2c to run before cwriter.Write(), because _WriteModuleInstanceImports now wants to read the .h output of wasm2c to know which module types are needed in the arguments to the instance init function. These are the "imported-from" modules